vulcan/lib/orchestration.rb

```diff
@@ -180,42 +180,44 @@ def run_as_copy_cell(storage:, build_target:, script:)
     def run_script_synchronously(storage:, build_target:, script:, token:)
       storage.with_run_cell_build(run_cell: storage.run_cell_for(build_target)) do |build_dir, output_files|
         result = run_script!(
-            storage: storage,
-            input_files: build_target.input_files,
-            output_files: output_files,
-            script: script,
-            token: token,
-            ch: build_target.cell_hash,
+          storage: storage,
+          input_files: build_target.input_files,
+          output_files: output_files,
+          script: script,
+          token: token,
+          ch: build_target.cell_hash,
         )
 
         if (error = result["error"])
           raise "Python error while executing script: #{error}"
         end
 
         storage.install_build_output(
-            buildable: build_target,
-            build_dir: build_dir
+          buildable: build_target,
+          build_dir: build_dir
         )
       end
     end
 
-      # Synchronous runner implementation that expects the transaction to complete.
+    # Synchronous runner implementation that expects the transaction to complete.
     def run!(storage:, build_target:, token:)
-      script = build_target.script
-      if run_as_copy_cell(storage: storage, build_target: build_target, script: script)
-        {'status' => 'done'}
-      elsif script
-        run_script_synchronously(
+      Yabeda.vulcan.job_runtime.measure do
+        script = build_target.script
```

Comment:
Here is an example where I 'measure' the time that vulcan jobs take to run and provide that as a histogram metric to prometheus.